src/Meshes.jl

```diff
@@ -26,7 +26,7 @@ import Tables
 import Random
 import Base: values, ==, +, -, *
 import StatsBase: sample
-import Distances: evaluate
+import Distances: colwise, evaluate
```

Comment:
I think we can move this to `using Distances: colwise` instead. The import is to extend the method table and we are not doing it in this PR.